plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -620,6 +803,7 @@ def get_transfer_transaction(  # pylint: disable=arguments-differ
         max_priority_fee_per_gas: Optional[str] = None,
         gas_price: Optional[str] = None,
         gas_price_strategy: Optional[str] = None,
+        gas_price_strategy_extra_config: Dict = {},
```

Comment:
Never set a dict or other mutable value in a constructor as default: http://pylint-messages.wikidot.com/messages:w0102

Here do instead:
```suggestion
        gas_price_strategy_extra_config: Optional[Dict] = None,
```